h2/src/main/org/h2/mvstore/db/MVTempResult.java

```diff
@@ -61,25 +61,39 @@ public void close() throws Exception {
      * Creates MVStore-based temporary result.
      *
      * @param database
-     *                        database
+     *            database
      * @param expressions
-     *                        expressions
+     *            expressions
      * @param distinct
-     *                        is output distinct
+     *            is output distinct
+     * @param visibleColumnCount
+     *            count of visible columns
      * @param sort
-     *                        sort order, or {@code null}
+     *            sort order, or {@code null}
```

Comment:
in general, I prefer the javadoc style where the comment is on the same line as the param name, mostly so I can see more code at once on my screen, but that's just a suggestion, not a requirement